src/components/Button.js

```diff
@@ -19,7 +19,8 @@ export const Button = ({
   onPress,
   small,
   smallest,
-  title
+  title,
+  whiteInvert = false
```

Comment:
i have difficulties to use this prop. this seems not very sustainable to me. hat if the surface is a different color than white? than this prop is wrong. `white` should not be part of the prop name.
couldn't we use the existing `invert` prop with an additional `lightest` or something like that?